habitat-lab/habitat/config/benchmark/rearrange/play_spot.yaml

```diff
@@ -23,6 +25,11 @@ habitat:
         disable_grip: False
         delta_pos_limit: 0.0125
         ee_ctrl_lim: 0.015
+      base_velocity_spot:
+        allow_dyn_slide: False
+        cylinder_deviate: 0.3
+        lin_collision_threshold: 0.04
```

Comment:
could you add short comments on what each of these parameters mean?

Reply:
Thank you Akshara! I will add the comments.